hw10/capital_letters.py

```diff
@@ -0,0 +1,11 @@
+class MyStr(str):
+
+    def __init__(self, text):
```

Comment:
To avoid of implementing __init__ here you can use `self.upper()` in the `__str__` method